app/controllers/review_bids_controller.rb

```diff
@@ -96,26 +103,49 @@ def set_priority
   def assign_bidding
     # sets parameters used for running bidding algorithm
     assignment_id = params[:assignment_id].to_i
+    assignment = Assignment.find(assignment_id)
     # list of reviewer id's from a specific assignment
```

Comment:
Huh?!  Which specific assignment?  Why are you gathering reviewer ids?  This code needs to be explained or rewritten.